docs/docs/hpc-on-k8s/genome.md

```diff
@@ -0,0 +1,62 @@
+# Genome Workload
+FastQC is a widely used tool for assessing the quality of genome sequence data. It provides a comprehensive overview of various quality metrics that are crucial for ensuring the reliability and accuracy of genomic data. 
+
+
+## Steps to Execute Job
+Below is the example manifest file for performing genome quality checks in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: fastqc-
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  parallelism: 1
+  completions: 1
+  suspend: true
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: openflighthpc/genome:latest
+        command: [ "/bin/bash", "-c", "python3 /app/script.py && echo '' && ls / && ls /app/ &&cat /app/sample_data_fastqc.html " ]
+        env:
+          - name: MINIO_IP
+            value: <minio ip like "10.151.15.78">
+          - name: MINIO_PORT
+            value: <minio port like "31100">
+          # Minio Access Key   
+          - name: MINIO_AKEY
+            value: <minio-access-key>
+          # Minio Secret Key   
+          - name: MINIO_SKEY
+            value: <minio-secret-key>
+          - name: BUCKET_NAME
+            value: <minio bucket name like "genome">
```

Comment:
Some issues experienced IRL with not quoting strings, perhaps putting the entire <> section in quotes would make this a little clearer (especially as the access key and secret key don't have any indication of this being needed)

Reply:
updated thanks